app/assets/stylesheets/navbar_left.scss

```diff
@@ -31,8 +31,6 @@
     }
   }
 
-  .selected, .selected a { color: $black; }
```

Comment:
You can have a deeper look into that file. Some other declaration are probably useless. I didn't clean it because I'm so lazy :)